exercises/practice/reverse-string/.approaches/introduction.md

```diff
@@ -0,0 +1,66 @@
+# Introduction
+
+There are various ways to solve **reverse-string**.
+This exercise has a clear exemplar solution and that is the built-in `reverse` method.
+However, this document will also explore other approaches.
+
+## General guidance
+
+The goal of this exercise is to reverse a string.
+
+## Approach: Using the built-in `reverse` method
+
+This is the most exemplary and straightforward solution.
+It uses the built-in [`reverse` method][reverse-method] to reverse the string.
+
+```crystal
+module ReverseString
+  def self.reverse(input : String) : String
+    input.reverse
+  end
+end
+```
```

Comment:
No separate approach page for this? Even if it is very simple, I would still recommend creating one.